playwright/_impl/_transport.py

```diff
@@ -161,6 +161,7 @@ def __init__(
 
     def request_stop(self) -> None:
         self._stopped = True
+        self.emit("close")
```

Comment:
I did not find a place that listens to this. Is it a bug fix?

Reply:
with the new disconnect handling, we use it to listen on the transport websocket disconnect via this

Reply:
Required for when the websocket connection is closed without client closing